### PR TITLE
Add users to db

### PR DESCRIPTION
**Pull request by itemniner**
Co-authored-by: Steven Meyers <sjjmeyers@gmail.com>
Co-authored-by: Jenny Klich <jennyklich@gmail.com>

## Type of change made:
- [ ] **Bugfix** (non-breaking change which fixes an issue)
- [x] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] **Refactoring** (improves existing functionality)
- [ ] **Testing** (tests existing code or improves existing tests)
- [ ] **Styling** (no functionality changed, just visual improvements)
## Description:
Add user models and end points to add users given params first name, last name, email, street address, state, zip and retrieve user by given path "/users" to get all users and "/user/:id" to get a specified user.
## What functionality does this accomplish?
- adds user to database. setup migration and adds end points for create, get all and get user <br>
closes # | does not close anything
## Checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code
- [ ] I have partially tested my code (please explain why):

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+app.config.from_object(os.environ['APP_SETTINGS'])
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(app)
+
+from models import User
+
+@app.route("/")
+def hello():
+    return "Hello World!"
+
```
Comment (SMJ289) [inline app.py:16]: Did we want to keep this Hello World test path?

File: app.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+@app.route("/getall")
+def get_all():
+    try:
+        users = User.query.all()
+        return jsonify([e.serialize() for e in users])
+    except Exception as e:
+	    return(str(e))
+
```
Comment (SMJ289) [inline app.py:49]: Should this path be `/users` and `/user/:id` to be more like what we have done in the past?

File: models.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    __tablename__ = 'users'
+
+    id = db.Column(db.Integer, primary_key=True)
+    first_name = db.Column(db.String())
+    last_name = db.Column(db.String())
+    email = db.Column(db.String(), unique=True)
+    street_address = db.Column(db.String())
+    state = db.Column(db.String())
+    zip = db.Column(db.String())
+
```
Comment (SMJ289) [inline models.py:13]: We might have to add twitter information later for that functionality.

File: app.py (hunk 1 above)
Comment (jklich151) [reply app.py:16]: Maybe we'll keep it there to make sure things are working and then remove it later?

**Pull request merged**